test/widgets/lightbox_test.dart

```diff
@@ -558,4 +601,51 @@ void main() {
       check(platform.position).equals(position);
     });
   });
-}
+
+  group('LightboxHero', () {
+    testWidgets('no hero animation occurs between different message list pages for same image', (tester) async {
+      final channel = eg.stream(streamId: eg.defaultStreamMessageStreamId);
+      final message = eg.streamMessage(stream: channel,
+        contentMarkdown: ContentExample.spoilerHeaderHasImage.html, topic: 'test topic');
```

Comment:
nit: This example seems a bit too atypical. How about using something like `imageSingle` to replicate a message that a user is more likely to encounter.